core/src/main/java/org/apache/accumulo/core/client/mapred/AbstractInputFormat.java

```diff
@@ -115,6 +117,21 @@ public static String getClassLoaderContext(JobConf job) {
     return InputConfigurator.getClassLoaderContext(CLASS, job);
   }
 
+  /**
+   * Sets connection information needed to communicate with Accumulo for this job
+   *
+   * @param job
+   *          Hadoop job instance to be configured
+   * @param connectionInfo
+   *          Connection information for Accumulo
+   * @since 2.0.0
+   */
+  public static void setConnectionInfo(JobConf job, ConnectionInfo connectionInfo) throws AccumuloSecurityException {
+    ConnectionInfoImpl info = (ConnectionInfoImpl) connectionInfo;
```

Comment:
Another spot where the type should be the interface not the impl.

Reply:
Fixed in 705d604759f9495cb